antenna-documentation/src/site/markdown/generators/HTML-report-generator-step.md

```diff
@@ -1,5 +1,12 @@
-# HTML report generator
-This workflow step generates a file `./target/antenna/3rdparty-licenses.html` containing a description of the licensing situation.
+# HTML attribution document generator
+This generator generates a plain html attribution document listing all found component versions together with
+the available license and copyright data as well as the component id in the package-url format. The complete
+license texts of all found licenses are in the appendix and linked to the components using the corresponding license.
+
+The file is plain html with a parallelstyle sheet. The idea is that the file can easily be adapted to the needs
```

Comment:
Typo: a space is missing between parallel and style sheet. Just wondering: Is 'parallel' the correct word? Maybe 'associated' or 'accompanying' ?